src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Draw {
+    private String color;
```

Comment:
this field should be protected so we can use it in child classes;
create constructor for it
```suggestion
    protected String color;
```